protocol-designer/src/components/organisms/ConfirmDeleteEntityInUseModal/index.tsx

```diff
@@ -36,7 +36,7 @@ export function ConfirmDeleteEntityInUseModal(
           <Flex
             justifyContent={JUSTIFY_END}
             gridGap={SPACING.spacing8}
-            padding={SPACING.spacing24}
+            padding={`0 ${SPACING.spacing24} ${SPACING.spacing24} ${SPACING.spacing24}`}
```

Comment:
since right and left are the same

```suggestion
            padding={`0 ${SPACING.spacing24} ${SPACING.spacing24} `}
```